### PR TITLE
tool to update components

### PR DESCRIPTION
**Pull request by deitch**
Some of our components - notably alpine versions - are falling behind. Since they live in a lot of different files, it is painful to update them.

I grabbed the `update-component-sha.sh` utility from linuxkit, updated it for our purpose, and installed it here.

One can do:

```
./tools/update-component-sha.sh --image alpine 3.13.1
```

And it will update all files in git that have `Dockerfile` in their name - so it includes our various `Dockerfile.in` but excludes anything not in git - to have the alpine tag changed to `3.13.1`.

Of course, afterwards, one must check the changes in and open a PR. Which is good.

### CODE REVIEW

**Comment by deitch** [comment]
Once this is merged in, I will bump our various versions of alpine to 3.13.

**Comment by deitch** [comment]
I would like to add docs for this, but where is the right doc file for it?

**Comment by deitch** [comment]
Yes, I am consciously ignoring yetus on this. It is being too difficult about a simple script.

**Review by rvs — APPROVED** [review]
The tools itself is fine @deitch -- but even when we merge it -- please hold off on

> Once this is merged in, I will bump our various versions of alpine to 3.13

This needs to be done very carefully and only post 6.x move

**Comment by deitch** [comment]
Sure @rvs ; that is why it is two distinct PRs, this one being just the tool to enable it.

**Pull request merged**
